spec/factories/rent.rb

```diff
@@ -1,4 +1,8 @@
 FactoryBot.define do
   factory :rent do
+    from    { Faker::Date.today }
+    to      { Faker::Date.forward(2) }
+    book    { 1 }
```

Comment:
You should use a new book instead of a plain id: `create(:book)` is a better way to ensure than you will always have a valid book assigned.

If you want an specific user or book on creation, you can later add it on a create call:

```rb
let(:book_of_user) { create(:book, user: custom_user) }
```

Reply:
ok